core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.ts

```diff
@@ -307,6 +311,7 @@ angular.module('oppia').component('settingsTab', {
           controller: 'ConfirmOrCancelModalController'
         }).result.then(function() {
           ExplorationRightsService.makeCommunityOwned();
+          $rootScope.$applyAsync();
```

Comment:
@kevintab95 it is used here.

Reply:
Hm, I think this is incorrect. `$rootScope.$applyAsync();` should be called when the promise returned by makeCommunityOwned method resolves.

Reply:
You may need to pass this as a callback that will be executed in the "then" block of the http call. Please double-check other `ExplorationRightsService` methods that have backend calls too.